packages/sdk-webos/src/deviceManager.ts

```diff
@@ -73,10 +73,9 @@ export const launchWebOSimulator = async (c: RnvContext, target: string) => {
     if (!fsExistsSync(ePath)) {
         return Promise.reject(`Can't find simulator at path: ${ePath}`);
     }
-    if (isSystemWin) {
+    if (isSystemWin || isSystemLinux) {
         return executeAsync(c, ePath, { detached: true, stdio: 'ignore' });
```

Comment:
@Marius456 as you already fixing this can you migrate these options into preset to keep consistency

```
return executeAsync(c, ePath, `ExecOptionsPresets.FIRE_AND_FORGET)
```

Reply:
Added preset SPINNER_FULL_ERROR_SUMMARY because linux doesn't exit command until sim is open. Because of that  FIRE_AND_FORGET doesn't show that sim is open in terminal.